pkg/scheduler/algorithm/predicates/utils.go

```diff
@@ -19,7 +19,7 @@ package predicates
 import (
 	"strings"
 
-	v1 "k8s.io/api/core/v1"
```

Comment:
Can we revert this change since there is nothing updates else in this file and it is not a restriction that we should import without `v1` prefix